test/mechanics/timeIntegration/RK4.cpp

```diff
@@ -0,0 +1,57 @@
+#include "BoostUnitTest.h"
+
+#include <iostream>
+#include <Eigen/Core>
+#include "mechanics/timeIntegration/RK4.h"
+
+class HarmonicOscillator
+{
+
+    double m_gam;
+
+public:
+    HarmonicOscillator(double gam)
+        : m_gam(gam)
+    {
+    }
+
+    //! Equation for damped harmonic oscillator represented as
+    //! first order system
+    void operator()(const Eigen::Vector2d& w, Eigen::Vector2d& dxdt, double t)
+    {
+        dxdt[0] = w[1];
+        dxdt[1] = -w[0] - 2. * m_gam * w[1];
+    }
+
+    //! Solution for damped harmonic oscillator at time t
+    //! Initial conditions [displacement, velocity] w0
+    double exactSolution(const Eigen::Vector2d& w0, double t)
```

Comment:
Can you provide a link to this exact solution for further reading? Wiki or something.